.lintrunner.toml

```diff
@@ -174,13 +174,42 @@ exclude_patterns = [
 command = [
     'python3',
     'tools/linter/adapters/grep_linter.py',
-    '--pattern=irange',
+    '--pattern=c10::irange',
     '--linter-name=irange',
-    '--error-name=using c10::irange, use std::views::iota instead',
-    '--replace-pattern=s/irange/std::views::iota/',
+    '--error-name=using c10::irange, use nvfuser::irange instead',
+    '--replace-pattern=s/c10::irange/nvfuser::irange/',
     """--error-description=\
         using c10::irange in nvfuser code base is deprecated, \
-        use std::views::iota instead\
+        use nvfuser::irange instead\
+    """,
+    '--',
+    '@{{PATHSFILE}}'
+]
+is_formatter = true
+
+
+[[linter]]
+code = 'irangeheader'
```

Comment:
A minor suggestion: is it not possible for one linter to serve two patterns? IMO, it's best to have all the irange related stuff into one irange linter for simplicity.

Reply:
I don't think `grep_linter.py` supports this.